monetarium/urls.py

```diff
@@ -1,25 +1,14 @@
-"""monetarium URL Configuration
-
-The `urlpatterns` list routes URLs to views. For more information please see:
-    https://docs.djangoproject.com/en/3.1/topics/http/urls/
-Examples:
-Function views
-    1. Add an import:  from my_app import views
-    2. Add a URL to urlpatterns:  path('', views.home, name='home')
-Class-based views
-    1. Add an import:  from other_app.views import Home
-    2. Add a URL to urlpatterns:  path('', Home.as_view(), name='home')
-Including another URLconf
-    1. Import the include() function: from django.urls import include, path
-    2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
-"""
+from django import contrib
 from django.contrib import admin
 from django.urls import include, path
 
 from monetarium.settings import ADMIN_ENABLED
 
 urlpatterns = [
         path("", include("apps.core.urls")),
+        path("users/", include("apps.users.urls")),
+        path("users/", include("django.contrib.auth.urls")),
```

Comment:
you have repeated code in different prs and repeated errors with that, if there is code to be done in other branches wait for the approval of a pr, this will generate future conflicts

if the branch it's called "resolve relationships", only do this stuff, avoid making changes that have nothing to do with the branch proposal

Reply:
thanks for the tip